README-QUIC.md

```diff
@@ -70,6 +70,23 @@ using simple TCP/TLS-like usage. Note that OpenSSL has no direct support for
 HTTP/3 so connecting to an HTTP/3 server should be possible but sending an
 HTTP/3 request or receiving any response data is not.
 
+### How can I create a QUIC server with OpenSSL?
+
+Starting with OpenSSL 3.5, you can create a QUIC server. The standard `openssl s_server` command does NOT support QUIC. Instead, use the dedicated QUIC server demo application:
+
+```shell
+$ ./demos/quic/server/server <port-number> <certificate-file> <key-file>
+```
+
+For example:
+```shell
+$ ./demos/quic/server/server 4433 server.pem server.key
+```
+
+Replace `server.pem` and `server.key` with your certificate and private key files.
+
+For production applications, see the [Demo-Driven Design (DDD)][DDD] demos under `doc/designs/ddd/` for server implementation examples.
```

Comment:
Could you please update this so it includes the current changes from PR #27230 so we can close that one and use just this PR?